src/test/a11y/mocks.ts

```diff
@@ -178,11 +170,27 @@ mock('fees/feesClient', {
   }
 })
 
-mock('response/draft/responseDraftMiddleware', {
-  'ResponseDraftMiddleware': {
-    retrieve: (req: express.Request, res: express.Response, next: express.NextFunction): void => {
-      res.locals.user.responseDraft = mockedResponseDraft()
-      next()
+mock('common/draft/draftMiddleware', {
+  'DraftMiddleware': {
+    requestHandler: (draftType: string) => {
+      return (req: express.Request, res: express.Response, next: express.NextFunction): void => {
+        const draft = new Draft<DraftDocument>()
+        switch (draftType) {
+          case 'claim':
+            draft.document = mockedDraftClaim()
```

Comment:
maybe not for this PR, but `mockedDraftClaim`, `mockedResponseDraft` and `mockCCJDraft` don't look consistent. [position of "Draft"]

Reply:
Good idea, done.